### DISCUSSION TITLE
VPS Egg (Linux/WIndows)

### DISCUSSION THREAD

**Discussion by OfficialJens** [pterodactyl/panel#5022] [General Community]
Does anyone know a VPS Egg that can handle Windows and Linux as os?
This should be possible, but I can't find anything right away.

**Comment by parkervcp** [comment]
There is no way that this can be done in pterodactyl. You would would need root access which is blocked in the containers.

**Comment by NickFox007** [comment]
Linux could be. I have edited some egg for this and it works but havent found any for win...
